src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -1,6 +1,31 @@
 <template>
 	<system-page>
 		<div class="page-content" style="max-width: 20rem;">
+			<div v-if="passwordless && strategicPartnerLogo">
+				<div class="tw-flex tw-justify-center tw-items-center tw-relative tw-mb-2">
+					<div
+						class="tw-w-14 tw-h-14 tw-flex tw-justify-center
+							tw-items-center tw-rounded-full tw-z-1 tw-bg-white tw--mr-2 tw-border
+							tw-border-white tw-border-4 logo"
+					>
+						<img
+							v-if="strategicPartnerLogo"
+							:src="strategicPartnerLogo.url"
+							:alt="strategicPartnerLogo.alt"
+							class="tw-w-full tw-h-full tw-object-contain"
+						>
+					</div>
+					<div
```

Comment:
We could keep the Kiva logo if the partner is missing. That would make this more reusable later...

Reply:
done